website/common/script/i18n.js

```diff
@@ -60,4 +61,26 @@ function t (stringName) {
   }
 }
 
+// translates passed strings to two (passed) languages and concatenates them
+// returns an array
+function translateTwice (languages, messages) {
```

Comment:
It's there for DRY, which is most evident in the cases where more than one translatable string is needed in construction. Example for explicit duplication: http://sprunge.us/iGaR?diff
But it's easy to go either way.